<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="mainHLayout">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QTabWidget" name="configTabHost">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="SingleScreenConfigPage" name="tab">
       <attribute name="title">
        <string>Single Screen</string>
       </attribute>
      </widget>
      <widget class="BacklightConfigPage" name="tab_2">
       <attribute name="title">
        <string>Backlight</string>
       </attribute>
      </widget>
      <widget class="RotatingBacklightConfigPage" name="tab_4">
       <attribute name="title">
        <string>Rotating Backlight</string>
       </attribute>
      </widget>
      <widget class="TripleScreenConfigPage" name="tab_3">
       <attribute name="title">
        <string>Triple Screen</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="startStopButton">
        <property name="text">
         <string>Let there be light</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>724</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionLED_Configuration"/>
    <addaction name="actionSerial_Configuration"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionLED_Configuration">
   <property name="text">
    <string>LED Configuration</string>
   </property>
  </action>
  <action name="actionSerial_Configuration">
   <property name="text">
    <string>Serial Configuration</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SingleScreenConfigPage</class>
   <extends>QWidget</extends>
   <header>singlescreenconfigpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TripleScreenConfigPage</class>
   <extends>QWidget</extends>
   <header>triplescreenconfigpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BacklightConfigPage</class>
   <extends>QWidget</extends>
   <header>backlightconfigpage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RotatingBacklightConfigPage</class>
   <extends>QWidget</extends>
   <header>rotatingbacklightconfigpage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
